src/pfs/fuse/fuse.go

```diff
@@ -0,0 +1,127 @@
+package fuse
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"bazil.org/fuse"
+	"bazil.org/fuse/fs"
+	"github.com/pachyderm/pachyderm/src/pfs"
+	"github.com/pachyderm/pachyderm/src/pfs/pfsutil"
+	"golang.org/x/net/context"
+)
+
+func Mount(apiClient pfs.ApiClient, repositoryName string, commitID string, mountPoint string) error {
+	if err := os.MkdirAll(mountPoint, 0777); err != nil {
+		return err
+	}
+	c, err := fuse.Mount(
+		mountPoint,
+		fuse.FSName("pfs"),
```

Comment:
I'll make these constants. These are just the strings that show up when you do `mount` from the command line.